common/src/Model/EntityRotationPolicy.cpp

```diff
@@ -198,27 +206,26 @@ namespace TrenchBroom {
 
                         // if the origin of the definition's bounding box is not in its center, don't apply the rotation
                         const auto offset = entity->origin() - entity->definitionBounds().center();
-                        if (offset.x() == 0.0 && offset.y() == 0.0) {
-                            if (entity->hasAttribute(AttributeNames::Angles)) {
-                                type = RotationType::Euler;
-                                attribute = AttributeNames::Angles;
-                            } else if (entity->hasAttribute(AttributeNames::Mangle)) {
-                                if (kdl::cs::str_is_equal(classname, "info_intermission")) {
-                                    type = RotationType::Euler_PositivePitchDown;
-                                } else {
-                                    type = RotationType::Mangle;
-                                }
-                                attribute = AttributeNames::Mangle;
-                            } else {
-                                type = RotationType::AngleUpDown;
-                                attribute = AttributeNames::Angle;
-                            }
+                        if (!(offset.x() == 0.0 && offset.y() == 0.0)) {
+                            // TODO: this only makes sense for Quake
```

Comment:
Here we have another case where it's game config dependent. In the long run, we will have to make this configurable via the game configuration I'm afraid.